etc/config.sample.toml

```diff
@@ -28,11 +28,15 @@ reporting-disabled = false
 ###
 ### [data]
 ###
-### Controls where the actual shard data for InfluxDB lives.
+### Controls where the actual shard data for InfluxDB lives and how it is
+### flushed from the WAL.
 ###
 
 [data]
   dir = "/var/opt/influxdb/data"
+  MaxWALSize = 104857600
```

Comment:
need comments for each of the following lines describing what they are and what the value is